spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/MessageSerializationUtils.java

```diff
@@ -45,122 +38,36 @@
 	 * Serialize the message payload unless it is a byte array.
 	 *
 	 * @param message the message with the payload to serialize
-	 * @param codec the codec used for serialization
 	 * @return the Message with teh serialized payload
 	 */
-	public static MessageValues serializePayload(Message<?> message, Codec codec) {
+	public static MessageValues serializePayload(Message<?> message) {
```

Comment:
I guess we no longer need this method or even simplify to return `MessageValues` at the higher level in the invocation stack.

Reply:
Yes that's the goal, just not yet for this PR giving it's size. We need to move header serdes into its own interceptor too, then we can remove this method completely, and probably Receiving and Sending Handlers too